test/helpers/mocks.json

```diff
@@ -11,7 +11,8 @@
     "img": "<img src=\"http://stat.ic/img.jpg\" alt=\"Image\"/>",
     "void": "<link/><meta/><img/><br/><hr/><input/>",
     "comment": "<!-- comment -->",
-    "doctype": "<!DOCTYPE html>"
+    "doctype": "<!DOCTYPE html>",
+    "svg": "<svg viewBox=\"0 0 512 512\" id=\"foo\">Inner</svg>"
```

Comment:
Do you mind moving this line under `svg` property?
```js
{
    "html": {
        // ...
    },
    "svg": {
        "simple": "<svg viewBox=\"0 0 512 512\" id=\"foo\">Inner</svg>",
        // ...
    }
}
```

I should also rename this file to `data.js` but you don't have to do it in this PR